e2e/runner/bitcoin.go

```diff
@@ -80,45 +89,69 @@ func (r *E2ERunner) GetTop20UTXOsForTssAddress() ([]btcjson.ListUnspentResult, e
 }
 
 // DepositBTCWithAmount deposits BTC into ZetaChain with a specific amount and memo
-func (r *E2ERunner) DepositBTCWithAmount(amount float64, memo *memo.InboundMemo) *chainhash.Hash {
+//
+// If `addDepositFee` is true, the depositor fee will be added to the amount, so that
+// the receiver will get the exact given 'amount' in ZetaChain
+func (r *E2ERunner) DepositBTCWithAmount(amount float64, memo *memo.InboundMemo, addDepositFee bool) *chainhash.Hash {
```

Comment:
Can we maybe create another function `DepositBTCWithExactAmount` that removes the last argument?
I believe you would use true almost all the time so better to have a simpler interface by default

Reply:
that's a good name!

Reply:
created a separate `DepositBTCWithExactAmount` method https://github.com/zeta-chain/node/pull/3358/commits/a0604ef8e5e77ee5253aaa3a3775388b36496a6f